src/EA.Weee.Core.Tests.Unit/Shared/PasteProcessorTests.cs

```diff
@@ -278,6 +281,39 @@ public void ParseNonObligatedPastedValues_GivenPastedDataAndExistingData_NonObli
             }
         }
 
+        [Fact]
+        public void ParseEvidencePastedValues_GivenPastedDataAndExistingData_EvidenceCategoryValuesShouldBeCorrectlyPopulated()
```

Comment:
not massively important but would you mind adding //assert //act and //arrange above each section

Reply:
Added :)